### DISCUSSION TITLE
Max supported JDK version

### DISCUSSION THREAD

**Discussion by IngeniousTom** [SquaretestLLC/Squaretest#26] [Q&A] [answered]
What is the latest version of JDK that Squarest test generator supports. I mean for example my project in IntelliJ IDEA works on JDK 17.
I can clearly see that Squaretest plugin works fine there and generates tests properly. What is the max JDK version I can use with it?
Thanks in advance!

**Comment by nate-dev** [comment, marked as answer]
Thank you for raising this question.
Squaretest supports all JDKs version 1.8 and higher.
Edit: Squaretest 1.7.5 and 1.8.5 support the new, Java 21 SequencedCollection types.
